packages/@aws-cdk/core/lib/stack-synthesizers/default-synthesizer.ts

```diff
@@ -266,6 +266,23 @@ export class DefaultStackSynthesizer implements IStackSynthesizer {
     }, [artifactId]);
   }
 
+  /**
+   * Create a copy of this synthesizer,
+   * with all of the fields the same,
+   * but not bound to any stack yet.
+   *
+   * @experimental
+   */
+  public bootstraplessCopy(): IStackSynthesizer {
```

Comment:
I wonder if it's not simpler to make the role ARN attributes public on this class, and just explicitly instantiate the bootstrapless synthesizer...

Reply:
Agreed.